packages/app-builder/src/components/Cases/Overview/Panel/AutoAssignmentPanelContent.tsx

```diff
@@ -0,0 +1,193 @@
+  const isInboxMember = (inbox: InboxWithCasesCount) => inbox.users.some((u) => u.userId === currentUserId);
+
+  // Check if user is admin of the inbox
+  const isInboxAdmin = (inbox: InboxWithCasesCount) =>
+    inbox.users.some((u) => u.userId === currentUserId && u.role === 'admin');
+
+  // Can edit inbox if global admin OR inbox admin
+  const canEditInbox = (inbox: InboxWithCasesCount) => hasEntitlement && (isGlobalAdmin || isInboxAdmin(inbox));
+
+  // Global admin sees all inboxes, others see only their inboxes (where they are a member)
+  const inboxes = isGlobalAdmin ? allInboxes : allInboxes.filter(isInboxMember);
+
+  // Can save if user has entitlement and can edit at least one inbox
+  const canSave = hasEntitlement && (isGlobalAdmin || inboxes.some(isInboxAdmin));
+
+  const handleToggleInbox = (inboxId: string, checked: boolean) => {
+    const inbox = inboxes.find((i) => i.id === inboxId);
+    const originalValue = inbox?.autoAssignEnabled ?? false;
+
+    setChanges((prev) => {
+      const newInboxes = { ...prev.inboxes };
+      if (checked === originalValue) {
+        delete newInboxes[inboxId];
+      } else {
+        newInboxes[inboxId] = checked;
+      }
+      return { ...prev, inboxes: newInboxes };
+    });
+  };
+
+  const handleToggleUser = (userId: string, checked: boolean) => {
+    const user = inboxes.flatMap((i) => i.users).find((u) => u.id === userId);
+    const originalValue = user?.autoAssignable ?? false;
+
+    setChanges((prev) => {
+      const newUsers = { ...prev.users };
+      if (checked === originalValue) {
+        delete newUsers[userId];
+      } else {
+        newUsers[userId] = checked;
+      }
+      return { ...prev, users: newUsers };
+    });
+  };
+
+  const handleSave = async () => {
+    setIsSaving(true);
+
+    try {
+      // Update inboxes
+      const inboxPromises = Object.entries(changes.inboxes).map(([inboxId, autoAssignEnabled]) => {
+        const inbox = inboxes.find((i) => i.id === inboxId);
+        if (!inbox) return Promise.resolve();
+
+        return updateInboxMutation.mutateAsync({
+          id: inboxId,
+          name: inbox.name,
+          escalationInboxId: inbox.escalationInboxId ?? null,
+          autoAssignEnabled,
+          redirectRoute: '/cases/overview',
+        });
+      });
+
+      // Update users
+      const userPromises = Object.entries(changes.users).map(([userId, autoAssignable]) => {
+        return editUserMutation.mutateAsync({
+          id: userId,
+          autoAssignable,
+        });
+      });
+
+      await Promise.all([...inboxPromises, ...userPromises]);
+
+      await queryClient.invalidateQueries({ queryKey: ['cases', 'inboxes'] });
+
+      toast.success(t('cases:overview.panel.auto_assignment.saved'));
+      closePanel();
+    } catch {
+      toast.error(t('cases:overview.panel.auto_assignment.save_error'));
+    } finally {
+      setIsSaving(false);
+    }
+  };
```

Comment:
Success/Failure toast should be managed by the server part using `toastSessionService`.

I don't think sending multiple requests (for inboxes update and users update) is a great thing.
In my opinion we should have only one remix endpoint for this, i.e:
`/ressources/cases/update-autoassign` which is taking body object with this kind of shape:
```
{
  inboxes: Record<uuid, boolean>,
  users: Record<uuid, boolean>,
}
```